### DISCUSSION TITLE
katana not working on Ubuntu 20.04.6 LTS

### DISCUSSION THREAD

**Discussion by d4715** [projectdiscovery/katana#677] [Q&A]
Hello There,

i want install tool called katana from project discovery but i can't 
i have to manually install this tool when i install tool manually so i download and copy katana file in [/root/go/bin/
![katana](https://github.com/projectdiscovery/katana/assets/11803332/53005d56-c8ed-4269-8089-94b8c4a83740)

then i just type this command katana --help i got these error   
(i attached screen shot)
katana: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by katana)
katana: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by katana)

i am using ubuntu (as a rdp server)
Distributor ID:	Ubuntu
Description:Ubuntu 20.04.6 LTS
Release:	20.04
Codename:	focal

so please help me out for this issues 

thanks

**Comment by MetzinAround** [comment]
Hi! You need to install the dependency GLIBC_2.34. https://ftp.gnu.org/old-gnu/Manuals/glibc-2.2.3/html_chapter/libc_36.html

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
